doc/README.md

```diff
@@ -252,7 +252,7 @@ run `pipenv shell` to enter pipenv virtual environment.
 
 You can use Visual Studio Code \(vscode\) to edit the code of Semgrep. The [reason-vscode](https://marketplace.visualstudio.com/items?itemName=jaredly.reason-vscode) Marketplace extension adds support for OCaml/Reason.
 
-The OCaml and Reason IDE extension by David Morrison is another valid extension, but it seems not as actively maintained as reason-vscode.
+The [OCaml and Reason IDE extension](https://github.com/reasonml-editor/vscode-reasonml) by @freebroccolo is another valid extension, but it seems not as actively maintained as reason-vscode.
```

Comment:
Actually i think now ocaml platform has a good official vs code mode. I didnt test it though

Reply:
I just wanted to fix the author's first name (Darin).